packages/shared/src/components/search/SearchContainer.tsx

```diff
@@ -42,14 +42,16 @@ export function SearchContainer({
       </main>
       <PageWidgets tablet={false} className="gap-6 items-center !px-0">
         <SearchFeedback className="order-2 laptop:order-1 max-w-widget" />
-        {!!content && (
+        {!!chunk && (
```

Comment:
Should we not evaluate if the chunk is the sources?

Reply:
The chunk is the whole object, this would just let us know if something is processing, so we can either display the loading state, or the list itself once the value is there. The component itself checks the sources list.